readthedocs/proxito/views/mixins.py

```diff
@@ -388,7 +393,7 @@ def canonical_redirect(
 
         log.info('Canonical Redirect.', host=request.get_host(), from_url=filename, to_url=to)
         resp = HttpResponseRedirect(to)
-        resp["X-RTD-Redirect"] = redirect_type or "unknown"
+        resp["X-RTD-Redirect"] = redirect_type.name
```

Comment:
It seems `redirect_type` can be `None`. In fact, it's the default value. If it's not allowed, we should remove the default and make it mandatory.